app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-<h1>List available users</h1>
\ No newline at end of file
+<section class="list-container">
+  <h1>Users</h1>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user) do %>
+      <div class="user-card">
+        <img src=<%=user.photo%> alt="user-profile" />
+        <div class="card-info">
+          <h2><%=user.name%></h2>
+          <p>Number of posts: <%= user.posts_counter %></p>
+        </div>
+      </div>
+    <% end %>
```

Comment:
- [ ] [OPTIONAL] Please consider creating `_user.html.erb` partial and rendering it as it makes your view files cleaner 👍 This would enable you have something like this 👉 ` <%= render @users %>`

Reply:
Sure ones I'm done with the basic layout I'll implement optional suggestions.